server/migrations/env.py

```diff
@@ -11,7 +11,7 @@
 
 # Interpret the config file for Python logging.
 # This line sets up loggers basically.
-fileConfig(config.config_file_name)
+fileConfig(config.config_file_name) # type: ignore
```

Comment:
This should probably be invesitgated and actually fixed, no?

Reply:
This is an automatically generated file. Therefore, I would advocate for not touching it.